docs/nodes/celestia-app-commands.md

```diff
@@ -153,6 +153,70 @@ the following command:
 celestia-appd tx gov vote <proposal id> <yes or no> --from <wallet> --chain-id <chain-id>
 ```
 
+Governance proposals on Celestia are limited as there are no text proposals,
+upgrades occur via social consensus, and some only params are not modifiable.
+However, one can submit a governance proposal to change certain parameters using
+the following instructions.
+
+To change the max validators to 105, one would first save this JSON file:
+
+```json
+{
+  "title": "Staking Param Change",
+  "description": "Update max validators",
+  "changes": [
+    {
+      "subspace": "staking",
+      "key": "MaxValidators",
+      "value": 105
+    }
+  ],
+  "deposit": "1000000000utia"
+}
+```
+
+Then you can submit the proposal with:
+
+```sh
+celestia-appd tx gov submit-legacy-proposal parameter-change <path to json file> --from <wallet> --chain-id <chain-id>
```

Comment:
What's the chain ID? Can you link to the possible values for different chains? Also link to how to choose/create wallets.

Reply:
sure updated in https://github.com/celestiaorg/docs/pull/1058/commits/4c1ecfabdfee3b708916bf26902f8cb838885405